doc/changes/changes_11.0.1.md

```diff
@@ -1,9 +1,21 @@
-# Virtual Schema Common JDBC 11.0.1, released 2023-??-??
+# Virtual Schema Common JDBC 11.0.1, released 2023-07-10
 
-Code name:
+Code name: Fix Issue With Integer Constants in `GROUP BY`
 
 ## Summary
 
+This release fixes an issue with queries using `DISTINCT` or `GROUP BY` with integer constants. Exasol interprets integers in `GROUP BY` clauses as column numbers which could lead to invalid results or the following error:
+
+```
+42000:Wrong column number. Too small value 0 as select list column reference in GROUP BY (smallest possible value is 1)
+```
+
+To fix this, VSCJ now replaces integer constants in `GROUP BY` clauses with a constant string. Please note that `GROUP BY <column-number>` is not affected by this and continues to work as expected.
```

Comment:
```suggestion
To fix this, VSCJ now replaces integer constants in `GROUP BY` clauses with a constant string.

Please that you can still safely use `GROUP BY <column-number>` in your original query, since Exasol internally converts this to `GROUP BY "<column-name>"`, so that the virtual schema adapter can tell both situations apart.
```